src/index.native.animated.js

```diff
@@ -71,6 +71,22 @@ class SwipeableViews extends Component {
      */
     onSwitching: React.PropTypes.func,
     /**
+     * Callback executed when the pan handler detects a touch-start
+     * event. Useful for nesting ScrollViews/SwipeableViews
+     *
+     * @param {object} event The raw event
+     * @param {object} gestureState The current gesture state
+     */
+    onTouchStart: React.PropTypes.func,
```

Comment:
That's a great idea to add them.
I have done it for the browser version. Could we follow the same pattern: https://github.com/oliviertassinari/react-swipeable-views/commit/c6fc0c24ced7d7d8170636b53d9a70f43b2c2514.